### PR TITLE
Selenium test to make sure shapefile option is shown in download tab f…

### PR DESCRIPTION
**Pull request by akashisama**
…or relevant layers

### CODE REVIEW

File: selenium/src/test/java/au/org/emii/portal/tests/step3/ShapeFileTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.util.List;
+
+public class ShapeFileTest extends BaseTest {
+
+    private static Logger log = Logger.getLogger(ShapeFileTest.class.getName());
+
+    @Test
+    public void verifyShapeFile() {
+        getDriver().get(AODN_PORTAL_SEARCH_PAGE + "?uuid=9d6b28d2-2d6b-4726-b603-ab03e3d15d34");
```
Comment (anguss00) [inline selenium/src/test/java/au/org/emii/portal/tests/step3/ShapeFileTest.java]: Hmm. May not be a biggie, but if this uuid dies for whatever reason, the test dies with it

Comment (anguss00) [inline selenium/src/test/java/au/org/emii/portal/tests/step3/ShapeFileTest.java]: Dunno how we get around that though...

Comment (akashisama) [inline selenium/src/test/java/au/org/emii/portal/tests/step3/ShapeFileTest.java]: Yeah there's no way around this for now. We can't find out via a selenium test which collections should have a shapefile, so we have to just pass it one manually.

Comment (anguss00) [inline selenium/src/test/java/au/org/emii/portal/tests/step3/ShapeFileTest.java]: Yup. Absolute worst case, test breaking is an alarm bell for this record being in a bad state

**Pull request merged**